README.md

```diff
@@ -869,124 +871,210 @@ After creating an initialization request successfully you can check at which sta
 
 ```json
 {
   "mode": "POPUP",
-  "redirect_url": "https://google.com",
-  "webhook_url": "https://webhook.site/7772560d-c97d-43cc-bf8e-248dc41946e1",
-  "purpose": "load agreement",
-  "phone": "<<PHONE>>",
-  "pan": "<<PAN_NUMBER>>",
-  "dob": "<<DOB>>",
-  "pdf_required": "Y",
-  "duration": 3
+  "webhook_url": "https://your-website.com/webhook",
+  "redirect_url": "https://your-website.com",
+  "purpose": "testing",
+  "phone": "9999999999",
+  "dob": "01-01-1990",
+  "phone_override": false,
+  "document_required": {
+    "ITR": 1,
+    "26AS": 2
+  },
+  "pan": "ABCDE1234F",
+  "pdf_required": false
 }
 ```
 
-| Parameters   | Mandatory | Description/Value                                    |
-| ------------ | --------- | ---------------------------------------------------- |
-| mode         | true      | REDIRECT or POPUP                                    |
-| redirect_url | false     | A valid URL                                          |
-| webhook_url  | true      | A valid URL                                          |
-| purpose      | true      | Your purpose                                         |
-| phone        | true      | Phone number linked to ITR portal                    |
-| pan          | true      | PAN number linked to ITR portal                      |
-| dob          | true      | Date of birth of the PAN holder in YYYY-MM-DD format |
-| pdf_required | false     | Whether you need PDF of ITR fetched                  |
-| duration     | false     | Years for which ITR details to fetch                 |
+| Parameters          | Mandatory | Description/Value                                      |
+| ------------------- | --------- | ------------------------------------------------------ |
+| mode                | true      | REDIRECT or POPUP                                      |
+| redirect_url        | false     | A valid URL                                            |
+| webhook_url         | true      | A valid URL                                            |
+| purpose             | true      | The reason for the transaction                         |
+| phone_override      | false     | If you wanted to update your phone number              |
+| document_required { | true      | If you wanted to update your phone number              |
+| ITR                 | false     | Specify for how many years of ITR you wanted to fetch  |
+| 26AS }              | false     | Specify for how many years of 26AS you wanted to fetch |
+| phone               | true      | Phone number you wanted to use at our platform         |
+| pan                 | true      | PAN number linked to ITR portal                        |
+| dob                 | true      | Date of birth of the PAN holder in MM-DD-YYYY format   |
+| pdf_required        | false     | Whether you need PDF of ITR fetched                    |
 
-<a name="itrRespParam"></a>
+**NOTE**: If you have provided `phone_override` to `true`, the user has to
```

Comment:
If a existing user (we can mention meaning of New and Existing User and reference those wherever we use the term) submits invalid combination of Phone registered against PAN and DOB. We will return an error that `phone number is invalid` (correct this message) with last 4 digits of correct phone number. 

In such scenario if the user doesn't have the phone number then they need to login into Income Tax Department (ITD) Portal using their ID password and remove us as e-Return Intermediary using this [process](http://add-the-pdf-url-here.com) 

Note: Update URL to instructions for removal, it can be a pdf or we can write another MD file. Take help from Business team to get the screenshots and business/design team to create a branded PDF will steps. Keep Akriti in loop.

Reply:
If an existing user (as explained below) submits invalid combination of Phone registered against PAN and DOB. We will return an error that 'phone number is invalid' (correct this message) with last 4 digits of correct phone number.

In such scenario if the user doesn't have the same phone number, then they need to login into Income Tax Department (ITD) Portal using their ID, Password and remove us as an e-Return Intermediary using this process (attach process link).

New User - New unique user being added to the ITD Portal for ITR and/or 26AS verification for the first time via Zoop.one

Existing User - User that was added to ITD by Zoop.one to verify and fetch ITR and/or 26AS details, has been requested for re-verification of details for the user.